applications/matter_weather_station/README.rst

```diff
@@ -16,6 +16,39 @@ You can use this application as a reference for creating your own application.
    The `Matter weather station application from the v2.1.1`_ |NCS| release participated in Matter Specification Validation Event (SVE) and successfully passed all required test cases to be considered as a device compliant with Matter 1.0.
    You can use the |NCS| v2.1.1 release to see the application configuration and the files that were originally used in Matter 1.0 certification.
 
+Overview
+********
+
+The application uses a single button for controlling the device state.
+The weather station device is periodically performing temperature, air pressure, and relative humidity measurements.
+The measurement results are stored in the device memory and can be read using the Matter controller.
+The controller communicates with the weather station device over the Matter protocol using Zigbee Cluster Library (ZCL).
+The library describes data measurements within the proper clusters that correspond to the measurement type.
+
+The application supports over-the-air (OTA) device firmware upgrade (DFU) using one of the two following protocols:
+
+* Matter OTA update protocol that uses the Matter operational network for querying and downloading a new firmware image.
+* Simple Management Protocol (SMP) over Bluetooth LE.
```

Comment:
```suggestion
* Simple Management Protocol (SMP) over Bluetooth® LE.
```

Reply:
Need to remove ® from Bluetooth® in the line 66.